lib/teleterm/vnet/service.go

```diff
@@ -542,9 +553,8 @@ func (p *clientApplication) OnNewConnection(ctx context.Context, appKey *vnetv1.
 
 		// Not passing ctx to ReportApp since ctx is tied to the lifetime of the connection.
 		// If it's a short-lived connection, inheriting its context would interrupt reporting.
```

Comment:
It seems to me `ReportSSHSession` should use the same approach wrt the context as `ReportApp`, no?

Reply:
yeah true, fixed, even though it was only used for a log.DebugContext call